### ISSUE TITLE
"odh directory" is not obvious in history or browser list. Add redirect?

### ISSUE THREAD

**Issue by WorldsEndless** [BYU-ODH/wp-humanities-byu#578]
there should be a page "directory" which redirects to "people" h

ttps://odh.byu.edu/people/

**Comment by nuevenueve** [comment]
I am not sure about the request. Would you like to create a empty directory page and then redirects its traffic to people page?
Thanks.

**Comment by WorldsEndless** [comment]
Yeah, that's right. Just an empty page at the /directory path that redirects to /people

**Comment by nuevenueve** [comment]
I could make a empty directory page and redirect it to the people page. 
Since it's about the redirect, do you want to change the people to directory instead?

**Comment by nuevenueve** [comment]
Okay, this is done with the meeting. :)

**Comment by WorldsEndless** [comment]
thank you, John!
